uaclient/cli.py

```diff
@@ -485,6 +485,58 @@ def refresh_parser(parser):
     return parser
 
 
+def magic_attach_parser(parser):
```

Comment:
If we get this into an API function, then I think subiquity will use that rather than a CLI subcommand.

We may want a user-facing CLI subcommand, but that probably won't require them to provide the ID. Instead, we'd automatically get that for them

Reply:
Indeed, once we have the API structure in place, I think we can retire this CLI entry. But for now, it will be useful for testing, like on integration tests